data/speakers.json

```diff
@@ -1475,13 +1475,13 @@
     "153": {
         "id": "153",
         "name": "Julien TOPCU",
-        "company": "Egencia & OWASP fundation",
+        "company": "Société Générale & OWASP fundation",
         "companyLogo": "/images/logos/egencia.jpg",
         "country": "France",
         "photoUrl": "images/speakers/Julien_TOPCU.png",
-        "shortBio": "Senior Lead Developer chez Egencia, je suis un fervent défenseur du Software Craftsmanship.",
-        "bio": "Senior Lead Developer chez Egencia, je suis un fervent défenseur du Software Craftsmanship. \n\nJ'évangélise activement autour de DDD/Hexagonal Architecture, l'XP et le Kanban #NoEstimates au sein de cours et de [meetups](https://www.meetup.com/fr-FR/Expedia-Inc-Paris/) que que je fais pour le compte d'Expedia, ainsi qu'à travers d'articles de blog.\n\nMembre de la fondation OWASP, je m'efforce de transmettre à la communauté une philosophie DevSecOps que je pense être l'avenir du métier de développeur. J'ai participé à la mise en place de pipeline de Continuous Security à l'échelles d'entreprises.",
-        "tags": [],
+        "shortBio": "Senior Lead Developer à la Société Générale, je suis un fervent défenseur du Software Craftsmanship.",
+        "bio": "Senior Lead Developer à la Société Générale, je suis un fervent défenseur du Software Craftsmanship. \n\nJ'évangélise activement autour de DDD/Hexagonal Architecture, l'XP et le Kanban #NoEstimates au sein de cours et de [meetups](https://youtu.be/P0_M00X4Xlw) ainsi qu'à travers d'articles de blog.\n\nMembre de la fondation OWASP, je m'efforce de transmettre à la communauté une philosophie DevSecOps que je pense être l'avenir du métier de développeur. J'ai participé à la mise en place de pipeline de Continuous Security à l'échelles d'entreprises.",
+        "tags": ["DevSecOps"],
```

Comment:
Hey @beyondxscratch 👋 
Thanks for the PR 👍 
There is no tag named `DevSecOps` but you can use the `Devops` tag. 👌 
I'll update the company logo asap.

Reply:
Hi @jlandure, can I use Security instead ?

Reply:
Of course! `Security`then 😄

Reply:
You can use both `Security` and `Devops`, it's an array.

Reply:
done